gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ espressoCore = "3.6.1"
 lifecycleRuntimeKtx = "2.8.7"
 activityCompose = "1.9.3"
 composeBom = "2024.10.01"
-maplibre = "11.6.1"
+maplibre = "11.7.1"
```

Comment:
had to bump this since there's no Vulkan at 11.6.1

Reply:
Yup; that makes sense! We wanted to upgrade to the latest anyways!